e2e_tests/e2e_test.py

```diff
@@ -13,11 +13,8 @@
 @pytest.fixture(scope='module')
 def test_app():
     # Create the new board & update the board id environment variable
-    trello_board = TrelloAPI().create_board("SeleniumTest")
-    os.environ['TRELLO_BOARD_ID'] = trello_board.board_id
-    # Use our test config instead of the 'real' version
-    file_path = find_dotenv('.env.test')
-    load_dotenv(file_path, override=True)
+    mongo_db = MongoWrapper().create_database("SeleniumTest")

```

Comment:
This relies on the environment variables being set (e.g. username/password etc) - we should try to load  in the `.env` file beforehand to ensure this is still working on local machines